nemo_text_processing/inverse_text_normalization/es/taggers/electronic.py

```diff
@@ -15,8 +15,18 @@
 import pynini
```

Comment:
Does this graph work on Instagram/Twitter handles @your_example_here? If not, could you make the alteration so we have better compatibility with web scraping?

Reply:
very valid use case! but this PR is specifically to enable capitalized inputs -- we can add that as a task in the backlog for when we can schedule general improvements

Reply:
Could you please add it as a future issue to keep consistency with other graphs being worked on?